src/stores/OriginalContractsStore.ts

```diff
@@ -1,24 +1,61 @@
 import { proxy } from 'valtio'
 import { subscribeKey } from 'valtio/utils'
 import WalletStore from 'stores/WalletStore'
+import defaultProvider from 'helpers/defaultProvider'
 import getOwnedERC721 from 'helpers/getOwnedERC721'
 
 interface ContractsStoreType {
-  contractsOwned: Promise<string[]>
+  contractsOwnedState: { [token: string]: number }
+  contractsOwnedStateAccount?: string
+  contractsOwned: string[]
+  lastBlockId?: number
+  locked: boolean
```

Comment:
Can we move the util state (e.g. `locked`) to a separate global variable? Don't want it to be a `proxy`